src/ui/AccountDropdown/UserInfo.svelte

```diff
@@ -66,7 +66,7 @@
     </div>
     <a
       class="upgrade btn-1 btn--orange btn--s mrg-m mrg--t v-center body-3"
-      href="https://app.santiment.net/pricing"
+      href="/pricing"
```

Comment:
Link should lead to `https://app.santiment.net/pricing` since it's might be accessed via insights' app 

```suggestion
      href="https://app.santiment.net/pricing"
```